stdlib/public/Concurrency/AsyncStream.swift

```diff
@@ -356,11 +356,6 @@ public struct AsyncStream<Element> {
 @available(SwiftStdlib 5.1, *)
 extension AsyncStream: AsyncSequence {
   /// The asynchronous iterator for iterating an asynchronous stream.
-  ///
-  /// This type doesn't conform to `Sendable`. Don't use it from multiple
```

Comment:
the 'this type isn't Sendable' assertion is still accurate, but i'm not sure it's particularly relevant or helpful to call out. the other warnings and references to fatalError() are no longer correct.

Reply:
I'd instead add _why_ it's not sendable. People sometimes ask about it, we should explain that so they don't assume it's just because it's "missing"

Reply:
excellent point, thank you.

> we should explain that so they don't assume it's just because it's "missing"

this is, in fact, my current assumption. i presume it's currently not Sendable mainly due to path-dependence/history – the original implementations intentionally did not support concurrent consumption, but then that restriction was lifted. the implementation of the relevant types appear to be effectively Sendable (basically everything gets forwarded from the public API into the various internal 'storage' types which appear generally thread-safe).